tests/e2e/cucumber/features/smoke/urlJourneys.feature

```diff
@@ -24,4 +27,15 @@ Feature: web can be navigated through urls
     And "Alice" shares the following resource using the direct url navigation
       | resource  | recipient | type | role     | resourceType |
       | lorem.txt | Brian     | user | Can view | file         |
+    When "Alice" opens the file "lorem.txt" through the URL
+    And "Alice" enters the text "Hello world" in editor "TextEditor"
+    And "Alice" saves the file viewer
+    And "Alice" closes the file viewer
```

Comment:
if you want to check file opening and you don't need change file content -> you can use existing steps
```
Then "Alice" is in a text-editor
And "Alice" closes the file viewer
```

Reply:
Yes, I just want to check the file opening. I'll refactor the step thanks!